test/run-e2e-tests.sh

```diff
@@ -16,12 +16,13 @@
 
 # This just run e2e tests.
 if [ -n "$KIND_E2E" ]; then
-    curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/v1.18.2/bin/linux/amd64/kubectl && chmod +x kubectl && sudo mv kubectl /usr/local/bin/
+    K8S_VERSION=${KUBERNETES_VERSION:-v1.18.2}
+    curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/${K8S_VERSION}/bin/linux/amd64/kubectl && chmod +x kubectl && sudo mv kubectl /usr/local/bin/
     wget https://github.com/kubernetes-sigs/kind/releases/download/v0.8.1/kind-linux-amd64
```

Comment:
Should be fine to keep kind version hard coded. Should support the last three versions at least.

Reply:
Yeah, that's what's currently hardcoded for the kind version https://github.com/kubernetes-sigs/descheduler/blob/master/.travis.yml#L16